packages/client-adapters/src/adapters/arethusa/adapter.js

```diff
@@ -99,7 +99,17 @@ class ArethusaTreebankAdapter extends BaseAdapter {
           this.addError(this.l10n.messages['MORPH_TREEBANK_UNSUPPORTED_LANGUAGE'].get(languageID))
           return
         }
-        const transformAdapter = new AlpheiosLexiconTransformer(this, new ImportData(languageModel, 'arethusa'))
+        let mapper = new ImportData(languageModel,'arethusa')
+        mapper.setPropertyParser(function (propertyName, propertyValue, inputElem) {
+          let propertyValues = []

```

Comment:
Are we using `propertyValues` anywhere?

Reply:
oh thanks for catching that, was from a prior iteration. Will fix before merge.